src/utils/recordedit-utils.ts

```diff
@@ -38,16 +38,23 @@ export function columnToColumnModel(column: any, queryParams?: any): RecordeditC
   );
   const logStackPathChild = column.isForeignKey ? LogStackPaths.FOREIGN_KEY : LogStackPaths.COLUMN;
 
+  let isRequired = !column.nullok;
+
   let type;
-  if (column.isAsset) {
+  if (column.isInputIframe) {
+    type = 'iframe';
+    // as long as any of the columns in the mapping are required, it should say its required
+    // isRequired
```

Comment:
I think you meant to update `isRequired` here by checking if any of the columns in the mapping are required.